hub2vu/aladin.py

```
@@ -0,0 +1,37 @@
+import requests
+from bs4 import BeautifulSoup
+url = "https://www.aladin.co.kr/shop/wbrowse.aspx?CID=1&BrowseTarget=List"
+response = requests.get(url)  # 요청 보내기
```

Ruff (0.13.1)

Probable use of `requests` call without timeout

(S113)